docs/toc.md

```diff
@@ -11,25 +11,9 @@
 
 ## [Known Issues](knownissues.md)
 
-# Controls
+# Universal Windows Platform (UWP)
 
-## Controls for WPF and Windows Forms
-
-### [InkCanvas](controls/wpf-winforms/InkCanvas.md)
-
-### [InkToolbar](controls/wpf-winforms/InkToolbar.md)
-
-### [MapControl](controls/wpf-winforms/MapControl.md)
-
-### [MediaPlayerElement](controls/wpf-winforms/MediaPlayerElement.md)
-
-### [WebView](controls/wpf-winforms/WebView.md)
-
-#### [WebView known issues](controls/wpf-winforms/WebView-known-issues.md)
-
-### [WebViewCompatible](controls/wpf-winforms/WebViewCompatible.md)
-
-### [WindowsXamlHost](controls/wpf-winforms/WindowsXamlHost.md)
+## Controls
 
 ## [AdaptiveGridView](controls/AdaptiveGridView.md)
```

Comment:
We need to make sure we indent the things underneath the controls category as well. 🙂